.bazeliskrc

```diff
@@ -0,0 +1 @@
+USE_BAZEL_VERSION=5.x
```

Comment:
Remove this and `.gitignore` changes (?)

Also OOC did you run into Bazel issues?

Reply:
Whoops, sorry this snuck in. Yes, Bazel 6.0 fails to build this, see #31504. Swapping to bazelisk to control bazel works just fine.